network/hnswrapper/hnsv2wrapperfake.go

```diff
@@ -328,6 +331,35 @@ func (fCache FakeHNSCache) SetPolicy(setID string) *hcn.SetPolicySetting {
 	return nil
 }
 
+func (fCache FakeHNSCache) PrettyString() string {
```

Comment:
Any reason not to make this implement `io.Stringer` directly?

Reply:
we decided against implementing `io.Stringer` in #1223

Reply:
Okay, the reasoning makes sense to me there.